File: Demo/Demo_ObjC/ViewController.m
//
//  ViewController.m
//  ObjC
//
//  Created by liguoliang on 2021/2/8.
//

#import "ViewController.h"

@interface ViewController ()
@property (nonatomic) UILabel *label;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"Demo_ObjC";
}

- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    self.label.text = [GLEnvs loadEnv][@"host"];
    [self.label sizeToFit];
    self.label.center = self.view.center;
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    NSLog(@"%@", [GLEnvs loadEnv]);
}

- (UILabel *)label {
    if(!_label) {
        _label = [UILabel new];
        [self.view addSubview:_label];
    }
    return _label;
}


@end

File: Demo/Demo_Swift/ViewController.swift
//
//  ViewController.swift
//  Demo_Swift
//
//  Created by liguoliang on 2021/2/8.
//

import UIKit
import GLEnvs

class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.title = "Demo_Swift"
        label.text = GLEnvs.loadEnv()["host"] as? String
        label.sizeToFit()
        label.center = self.view.center
    }
    
    lazy var label:UILabel = {
        let label = UILabel()
        self.view.addSubview(label)
        return label
    }()
}

